src/LandIce/problems/LandIce_ParamEnum.hpp

```diff
@@ -13,7 +13,10 @@ enum class ParamEnum
   MuCoulomb    = 2,
   MuPowerLaw   = 3,
   Power        = 4,
-  Homotopy     = 5
+  Homotopy     = 5,
+  Kappa_x      = 6,
+  Kappa_y      = 7,
+  Kappa_z      = 8
 };
```

Comment:
Why are you using this for a non LandIce problem? You could use PHAL_SharedParameter instead of the LanIce_Shared Parameter.

Reply:
I couldn't get it to work for unknown reasons (likely lack of experience on my part) - see my other reply.  What I could do it remove the "Kappa_x", "Kappa_y", "Kappa_z" enums and hijack the others that were already there, as Luca suggested.  Not sure if this would cause more or less confusion.